modules/wpml/wpml-config.php

```diff
@@ -262,6 +279,81 @@ public function translate_taxonomies( $taxonomies, $hide ) {
 		return $taxonomies;
 	}
 
+	/**
+	 * Translation management for strings in blocks content.
+	 *
+	 * @since 3.3
+	 *
+	 * @param array[] $parsing_rules Rules as Xpath expressions to evaluate in the blocks content.
+	 * @return array[] Rules completed with ones from wpml-config file.
+	 *
+	 * @phpstan-param array<string,array<string>> $parsing_rules
+	 * @phpstan-return array<string,array<string>> $parsing_rules
+	 */
+	public function translate_blocks( $parsing_rules ) {
+		return $this->extract_blocks_parsing_rules( $parsing_rules, 'xpath' );
+	}
+
+	/**
+	 * Translation management for blocks attributes.
+	 *
+	 * @since 3.3
+	 *
+	 * @param array[] $parsing_rules Rules for block attributes to translate.
```

Comment:
We may need to choose between **blocks attributes** and  `block attributes` for consistency.

Reply:
Done in 929a19365d65402d6b51bd655f0ba16f79aabfd0

Reply:
This is not the same in the name of the method just below.

Reply:
Done in 5c8c4da5fd920a3c525e8bbaa1d90f4327c95f17